site/content/internal/gitlab-omnibus.md

```diff
@@ -0,0 +1,247 @@
+---
+title: GitLab Omnibus
+heading: "GitLab Omnibus Release Process"
+description: "Information on updating and building GitLab Omnibus with new versions of Mattermost"
+date: 2021-03-12T14:59:29-05:00
+section: internal
+weight: 130
+---
+
+GitLab's Omnibus package bundles Mattermost TE as an optional feature that can be enabled during installation. While GitLab maintains most of this integration, we send them new versions of Mattermost and occasionally assist with support on issues that relate to Mattermost.
+
+For every monthly GitLab release, we submit a merge request (MR) to [GitLab's repository](https://gitlab.com/gitlab-org/omnibus-gitlab/) to update the embedded version of Mattermost. GitLab releases in the middle of the month, so we'll generally submit the newest version of Mattermost to them at the start of the month to give time for the review process to happen.
+
+### Setting up GitLab Omnibus for development
+
+We maintain [our own fork of GitLab Omnibus](https://gitlab.com/mattermost/omnibus-gitlab) for use when submitting merge requests upstream. This should be cloned and kept up to date to ensure that we're testing and submitting against the latest code.
+
+These are the steps for checking out GitLab Omnibus and installing its dependencies:
+
+1. Clone our fork.
+
+    ```bash
+    git clone https://gitlab.com/mattermost/omnibus-gitlab.git
+    ```
+       
+2. Add the upstream repository as a git remote.
+
+    ```bash
+    git remote add upstream https://gitlab.com/gitlab-org/omnibus-gitlab.git
+    ```
+
+3. Install Docker. GitLab packages all dependencies for building GitLab Omnibus in a Docker image which will be pulled later.
+
+### Submitting a new version of Mattermost
+
+These are the steps to update GitLab Omnibus with a new version of Mattermost:
+
+1. Ensure our fork's master branch is up to date with upstream.
+
+    ```bash
+    git checkout master
+    git fetch upstream
+    git pull upstream master
+    git push
+    ```
+       
+1. Create a branch for the new version of Mattermost.
```

Comment:
nit: I know this is handled at render time, but can we increment the list here for easy "plaintext" reading too?